src/main/java/com/squareup/catalog/demo/Main.java

```diff
@@ -233,29 +270,26 @@ private void executeExample(String exampleName, boolean cleanup, CatalogApi cata
   }
 
   /**
-   * Attempts to log {@link Error}s from an {@link ApiException}, or rethrows if the response body
-   * cannot be parsed.
+   * Finds out the currency to be used across all examples Use join() because we
+   * can't proceed further until we have this information.
```

Comment:
`Use join() because we can't proceed further until we have this information.`
What does this mean?

Reply:
I think it's worth mentioning that this step could be hardcoded instead if you know your app will only be used in a specific country/currency.

Reply:
`.join()` just means blocking until the result is here, as we can't move forward with processing the examples until we have currency to use. If this call fails we shouldn't continue. I'll rephrase the comment and add information about how this step can be hardcoded instead if you know your currency.